src/lib/noop.js

```diff
@@ -0,0 +1,14 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+// Simple helper functions
+// none of these need any external deps
+
+module.exports = function noop() {};
```

Comment:
I thought this was about the shortest possible module one could make... but I suppose you could do:
```js
exports.a = 0;
```
Kind of silly... but needed to de-circularize, short of refactoring `Lib` (see https://github.com/plotly/plotly.js/issues/236 )